catalogs/sources/gtfs/schedule/ie-munster-county-cork-west-cork-gtfs-2875.json

```diff
@@ -0,0 +1,28 @@
+{
+  "mdb_source_id": 2875,
+  "data_type": "gtfs",
+  "provider": "West Cork Connect",
+  "location": {
+    "country_code": "IE",
+    "bounding_box": {
+      "minimum_latitude": null,
+      "maximum_latitude": null,
+      "minimum_longitude": null,
+      "maximum_longitude": null,
+      "extracted_on": "2025-11-17 18:21:16.026797+00:00"
+    },
+    "subdivision_name": "Munster",
+    "municipality": "County Cork"
+  },
```

Comment:
It might be worth seeing if this data has been made available via the Irish NAP: https://developer.nationaltransport.ie/

If not, it would be good for us to reach out to them about it once this PR is submitted to let them know.

Reply:
In that case I would still add this as a separate feed then